schema_based_on_diagram.sql

```diff
@@ -0,0 +1,52 @@
+/* CREATE TABLES BASED ON DIAGRAM */
+
+/* create patients table */
+CREATE TABLE patients ( 
+    id INT PRIMARY KEY NOT NULL, 
+    name VARCHAR(25) NOT NULL, 
+    date_of_birth DATE NOT NULL 
+    );
+
+/* create medical_histories table */
+CREATE TABLE medical_histories (
+	id INT PRIMARY KEY NOT NULL, 
+	patient_id INT, 
+    CONSTRAINT fk_patient
+  	FOREIGN KEY(patient_id)
+  		REFERENCES patients(id)
+);
+
+/* create treatments table */
+CREATE TABLE treatments (
+	  id INT PRIMARY KEY NOT NULL,
+	   type VARCHAR(255), 
+	   name VARCHAR(255)
+);
```

Comment:
As of the [project requirement ](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md#project-requirements) kindly, create a new `join table` for the `many-to-many` tables in this case for `medical_histories` and `treatments` that will have a column for the primary key of both tables as reference.

<img width="940" alt="image" src="https://user-images.githubusercontent.com/34578631/172586199-19d40fba-c718-401e-8c1b-b8fe24f37af0.png">